v2/internal/genericarmclient/suite_test.go

```diff
@@ -33,7 +34,9 @@ func setup() error {
 	format.TruncateThreshold = 4000 // Force a longer truncate threshold
 
 	// setup global logger for controller-runtime:
-	ctrl.SetLogger(klogr.New()) //nolint: staticcheck
+	cfg := textlogger.NewConfig(textlogger.Verbosity(3)) // Use verbose logging in tests
```

Comment:
Should we be using our predefined debug log-levels here rather than hardcoding 3?

Reply:
Done.